doc/source/data/package-ref.rst

```diff
@@ -122,6 +122,21 @@ Custom Datasource API
 .. autoclass:: ray.data.ReadTask
     :members:
 
+Datasource Partitioning API
+---------------------
+
+.. autoclass:: ray.data.datasource.PartitionStyle
+    :members:
+
```

Comment:
```suggestion

.. autoclass:: ray.data.datasource.PathPartitionScheme
    :members:
    
```
(unless there's a reason for `PathPartitionScheme` to be intentionally omitted)

Reply:
Fixed!